### ISSUE TITLE
Doc Review: VCH Status Reference

### ISSUE THREAD

**Issue by anchal-agrawal** [vmware/vic#3356]
https://github.com/vmware/vic/blob/master/doc/user_doc/vic_admin/vicadmin_status_ref.md

**Comment by anchal-agrawal** [comment]
This [heading](https://github.com/vmware/vic/blob/master/doc/user_doc/vic_admin/vicadmin_status_ref.md#vch-vch) should say **Virtual Container Host (VCH)**.

> VCH Admin checks that the firewall is correctly configured on the ESXi host or the ESXi hosts in the cluster on which the VCH is running.
VCH Admin checks that the firewall is correctly configured on the ESXi **host(s)** in the cluster on which the VCH is running.

**Comment by sgairo** [comment]
The hyperlink ,  VCH Deployment Fails with Firewall Validation Error, takes to a 404 within Github,  has that page been made yet?

**Comment by anchal-agrawal** [comment]
That page exists. That hyperlink should go to a .md, not .html.

I think this is due to https://github.com/vmware/vic/issues/2614#issuecomment-252865439. @stuclem?

**Comment by stuclem** [comment]
Yes, @anchal-agrawal that's right - it's a shortcoming of gitbook that linking across books doesn't work, so I have to include the link to the HTML rather than to the MD.

**Comment by stuclem** [comment]
Thanks @anchal-agrawal and @sgairo!

- Fixed the heading to reinstate Virtual Container Host (this accirdentally got zapped by a global search and replace of "Virtual Container Host" to "VCH", done because I thought that the full version was making things too wordy)
- For the firewall, I was trying to convey the standalone/cluster deployments. Reworded as "VCH Admin checks that the firewall is correctly configured on a an ESXi host on which the VCH is running. If the VCH is running in a cluster, VCH Admin checks the firewall configuration on all of the hosts in the cluster."

Changes are in https://github.com/stuclem/vic/commit/ae482cf805786b6a82ec44804675b688c283073d.
Is this OK now?

**Comment by sgairo** [comment]
LGTM :)

**Comment by anchal-agrawal** [comment]
Thanks Stuart! There's an extra "a" in "VCH Admin checks that the firewall is correctly configured on **a an ESXi** host on which the VCH is running."

LGTM

**Comment by stuclem** [comment]
Fixed. Good spot @anchal-agrawal :)

**Comment by stuclem** [comment]
LGTMs received from both reviewers. Closing.
